### PR TITLE
Added new workflow to update base branch

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/47323

## Description
A few sentences describing the overall goals of the pull request's commits.

### CODE REVIEW

**Comment by Shellyber** [comment]
Example workflow run on my forked repo (with the relevant changes to make this work) - https://github.com/Shellyber/content/runs/6259506423?check_suite_focus=true

File: Utils/github_workflow_scripts/sync_contrib_base.py [hunk 1]
```diff
@@ -54,14 +74,14 @@ def main():
     content_repo = gh.get_repo(f'{organization}/{repo}')
 
     master_sha = get_master_commit_sha(content_repo)
+    if ref_branch:
+        update_branch(content_repo, ref_branch, master_sha)
+        return
+
     contrib_base_branches = get_branch_names_with_contrib(content_repo)
+    print(f'updating {contrib_base_branches=}')
     for branch_name in contrib_base_branches:
```
Comment (ShahafBenYakir) [inline Utils/github_workflow_scripts/sync_contrib_base.py:83]:
```suggestion
    if ref_branch:
        # Case this flow was triggered on a specific branch
        contrib_base_branches = [ref_branch]
    else:
        # Case we are running scheduled job - detect all contrib/ base branches.
        contrib_base_branches = get_branch_names_with_contrib(content_repo)
    
    print(f'Updating {contrib_base_branches=}')
    
    for branch_name in contrib_base_branches:
```

**Review by ShahafBenYakir — COMMENTED** [review]
nice

**Review by ShahafBenYakir — APPROVED** [review]
Nice

**Pull request merged**